README.md

```diff
@@ -1,11 +1,13 @@
-# Time Switch
+# Time Switch - A very simple timer for your needs
 ![](https://github.com/fsobolev/timeswitch/raw/master/data/icons/hicolor/scalable/apps/io.github.fsobolev.TimeSwitch.svg)
-<br/>[<img src="https://flathub.org/assets/badges/flathub-badge-en.png" width=128px>](https://flathub.org/apps/details/io.github.fsobolev.TimeSwitch)
-<br/>[<img src="https://camo.githubusercontent.com/874898488d74b24f916891c19c132f4bae397ab1de4898cd25d268ec81c7d92b/68747470733a2f2f692e696d6775722e636f6d2f3049746a6f374e2e706e67" width=128px>](https://matrix.to/#/#sable-burrow:matrix.org)
+[<img src="https://flathub.org/assets/badges/flathub-badge-en.png" width=256px>](https://flathub.org/apps/details/io.github.fsobolev.TimeSwitch)
+[<img src="https://camo.githubusercontent.com/874898488d74b24f916891c19c132f4bae397ab1de4898cd25d268ec81c7d92b/68747470733a2f2f692e696d6775722e636f6d2f3049746a6f374e2e706e67" width=164px>](https://matrix.to/#/#sable-burrow:matrix.org)
+### **Time Switch** is a program that allows you to power off, reboot or suspend your system, send a notification (optionally with a sound) or execute any command/script on timer.
 
-Power off, reboot, suspend, send a notification or execute any command on timer.
+<p align="center">
+<img src="data/screenshots/main.png" width=180px>
+<img src="data/screenshots/command.png" width=180px>
+<img src="data/screenshots/running.png" width=180px>
+</p>
 
-<img src="data/screenshots/main.png" width=180px><img src="data/screenshots/command.png" width=180px><img src="data/screenshots/running.png" width=180px>
-
-**Time Switch** is a simple program that allows you to power off, reboot or suspend your system, send a notification (optionally with a sound signal) or execute any command on timer.
-The app is built for GNOME and uses LibAdwaita, but it works in any desktop environment.
+<sub>The app is built for GNOME using GTK4 and LibAdwaita, but it works under any desktop environment thanks to the flatpak technology.</sub>
```

Comment:
I don't see how current README is hard to read (it looks fine on all my devices) and how this one is any better: the name is as big as a short description, buttons are gigantic, last paragraph is barely noticeable compared to everything else ([screenshot](https://i.imgur.com/PeC6TyC.png)).

Reply:
[This](https://i.imgur.com/Y80z1MI.jpg) is how it looks on a phone